packages/hebao_v1/contracts/base/BaseWallet.sol

```diff
@@ -151,10 +147,6 @@ contract BaseWallet is ReentrancyGuard, AddressSet, Wallet
     {
         require(_method != bytes4(0), "BAD_METHOD");
         require(methodToModule[_method] == address(0), "METHOD_BOUND_ALREADY");
-        require(
-            controller.moduleRegistry().isModuleRegistered(_module),
```

Comment:
Maybe we should keep this one.

Reply:
`bindMethod` is also used for unbinding methods, so I removed it here as well to allow modules to be completely removed. Because the module will probably be completely setup, with all methods necessary bound, when it's added I don't think it matters much.